Dockerfile

```diff
@@ -0,0 +1,23 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Use the latest python image from DockerHub (3.x)
+FROM python:latest
```

Comment:
I would specify at least the major version of Python if not also the minor version, so that it is clear in the Dockerfile which Python version you are using.